### PR TITLE
Feature CPLP-727 Company detail overlay implemented

### PR DESCRIPTION
**Pull request by fatihayyildiz**
Registration requests table company detail overlay implemented

### CODE REVIEW

File: cx-portal/src/assets/locales/de/main.json [hunk 1]
```diff
@@ -50,7 +50,12 @@
         "buttonconfirm": "bestätigen",
         "buttondecline": "ablehnen",
         "cellconfirmed": "bestätigt",
-        "celldeclined": "abgelehnt"
+        "celldeclined": "abgelehnt",
+        "overlay": {
+          "title": "Firmendetails",
+          "companydatatitle": "Firmen Data",
```
Comment (oyo) [inline cx-portal/src/assets/locales/de/main.json:56]: Firmendaten

File: cx-portal/src/assets/locales/de/main.json [hunk 2]
```diff
@@ -50,7 +50,12 @@
...
+          "taxid": "Steuer ID"
```
Comment (oyo) [inline cx-portal/src/assets/locales/de/main.json:57]: Steuernummer

**Review by oyo — APPROVED** [review]
tested and everything looks good - only see my comments on the german texts. Plz fix and then directly merge :)

**Comment by fatihayyildiz** [comment]
> tested and everything looks good - only see my comments on the german texts. Plz fix and then directly merge :)

German texts fixed and ready for merge

**Pull request merged**